docs/v3/source/includes/resources/deployments/_header.md

```diff
@@ -16,4 +16,4 @@ Deployment strategies supported:
 applications to be deployed without incurring downtime by gradually rolling out instances. Max-in-flight can be configured
 to specify how many instances are rolled out simultaneously.
 
-* Canary deployments deploy a single instance and pause for user evaluation. If the canary instance is deemed successful, the deployment can be resumed via the [continue action](#continue-a-deployment). The deployment then continues like a rolling deployment. This feature is experimental and is subject to change.
+* [Canary deployments](https://docs.cloudfoundry.org/devguide/deploy-apps/rolling-deploy.html#canary-deployment-process) deploy a single instance and pause for user evaluation. If the canary instance is deemed successful, the deployment can be resumed via the [continue action](#continue-a-deployment). The deployment then continues like a rolling deployment. This feature is experimental and is subject to change.
```

Comment:
this link wont work until https://github.com/cloudfoundry/docs-dev-guide/pull/537/files is merged